### DISCUSSION TITLE
Prevent Duplicate Names using _uvc

### DISCUSSION THREAD

**Discussion by the4arms** [cortexrd/Knack-Toolkit-Library#153] [Q&A]
I'm trying to use the _uvc keyword to prevent duplicate names.  As you can see from the screen shots, I feel I have everything set-up correctly.  The _hv works but when I click the "submit" button, nothing happens.  Its like the page doesn't recognize the submit button.

Can someone help me?

Thanks in advance
<img width="1026" alt="Screenshot 2023-11-17 at 9 06 36 AM" src="https://github.com/cortexrd/Knack-Toolkit-Library/assets/151178940/fdad14cd-8e32-401d-9ccc-879f50a88e6e">
<img width="602" alt="Screenshot 2023-11-17 at 9 06 08 AM" src="https://github.com/cortexrd/Knack-Toolkit-Library/assets/151178940/7665c4b8-348e-4c3d-b3c6-2cd2ca4625a0">

**Comment by the4arms** [comment]
I figured it out .. I needed to make it  _uvc=First, " ",  Last

**Comment by cortexrd** [comment]
Good job!!
I admit it's not super intuitive at first.

I'm always happy to see it when people use those features.  The _uvc and _uvx keywords were quite complex to write.

Cheers,
Norm

**Reply by the4arms** [reply]
Thanks Norm!

Another questions .. how do I get the error message to not show up after the form has been submitted?  See attached picture

Thanks
Diann 
<img width="861" alt="Screenshot 2023-11-20 at 10 29 13 AM" src="https://github.com/cortexrd/Knack-Toolkit-Library/assets/151178940/07357dee-7b99-4f79-8676-5d04410e0008">

**Comment by cortexrd** [comment]
Hi Diann,

This is unexpected... it shouldn't happen.

Can you give me more information?  Was it a duplicate that was trapped as expected, or not?

If you could give me access to your app, it would be much easier.  Can you create me an account for testing?  No need to see any real data.  As long as I can reproduce the problem.

If that's ok, send me the login info at nd@ctrnd.com

Cheers,
Norm
